build.sh

```diff
@@ -127,11 +126,14 @@ if (( ${NUMARGS} == 0 )) || hasArg libcuml || hasArg prims || hasArg bench; then
     cd ${LIBCUML_BUILD_DIR}
 
     cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
+          -DNCCL_PATH=${INSTALL_PREFIX} \
+          -DWITH_UCX=OFF \
```

Comment:
probably move this option to after `-DBUILD_CUML_STD_COMMS=ON` just for visual clarity and ease of understanding